scripts/operations/validation/ncn1.3.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
The script should not be named after the section of the CSM validation where it is currently being run. Section numbers will change. The script name should describe what it is doing.